conans/client/conf/config_installer.py

```diff
@@ -116,8 +116,13 @@ def _process_download(item, cache, output, tmp_folder, verify_ssl, requester):
         raise ConanException("Error while installing config from %s\n%s" % (item, str(e)))
 
 
-def configuration_install(path_or_url, cache, output, verify_ssl, requester,
-                          config_type=None, args=None):
+def configuration_install(path_or_url,
+                          cache,
```

Comment:
No, please, use the full length of the line, we don't use this format for functions declarations :)

Reply:
this format remind me old C codes :joy: